gui/tabs/players_tab.cpp

```diff
@@ -41,44 +41,42 @@ namespace PlayersTab {
 				if (State.selectedPlayerId > -1 && GetPlayerData(GetPlayerControlById(State.selectedPlayerId)) != nullptr)
 				{
 
-					if (IsInMultiplayerGame() && !IsHost()) {
-						if (!GetPlayerData(*Game::pLocalPlayer)->fields.IsDead) {
-							if (ImGui::Button("Call Meeting")) {
-								PlayerControl_CmdReportDeadBody(*Game::pLocalPlayer, NULL, NULL);
-							}
-							ImGui::NewLine();
-							if (State.selectedPlayerId > -1 && State.selectedPlayer != *Game::pLocalPlayer && ImGui::Button("Report Body")) {
-								PlayerControl_CmdReportDeadBody(*Game::pLocalPlayer, GetPlayerData(State.selectedPlayer), NULL);
-							}
+					if (!GetPlayerData(*Game::pLocalPlayer)->fields.IsDead) {
+						if (ImGui::Button("Call Meeting")) {
+							State.rpcQueue.push(new RpcReportPlayer(*Game::pLocalPlayer, NULL));
+						}
+						ImGui::NewLine();
+						if (State.selectedPlayerId > -1 && State.selectedPlayer != *Game::pLocalPlayer && ImGui::Button("Report Body")) {
```

Comment:
need to make sure that the selected player that we are attempting to report is dead.